cmd/root.go

```diff
@@ -1144,12 +1146,16 @@ func runSignalWrapper(cmd *Command) (err error) {
 	switch {
 	case errors.Is(err, errSigInt):
 		cmd.logger.Infof("SIGINT signal received. Shutting down...")
+		time.Sleep(cmd.conf.WaitBeforeClose)
 	case errors.Is(err, errSigTerm):
 		cmd.logger.Infof("SIGTERM signal received. Shutting down...")
+		time.Sleep(cmd.conf.WaitBeforeClose)
 	case errors.Is(err, errSigTermZero):
 		cmd.logger.Infof("SIGTERM signal received. Shutting down...")
+		time.Sleep(cmd.conf.WaitBeforeClose)
 	case errors.Is(err, errQuitQuitQuit):
 		cmd.logger.Infof("/quitquitquit received request. Shutting down...")
+		time.Sleep(cmd.conf.WaitBeforeClose)
```

Comment:
Shall we just add one of these right above the return statement below?

Reply:
No, because we don't want to delay in the case of the default error.